vars/kubectl.groovy

```diff
@@ -57,6 +57,42 @@ String getSecretValue(String namespace, String secret_name, String key_name) {
     }
 }
 
+String createSecretWithJson(String secret_name, String json_value, String key_name, String namespace) {
+    try {
+        sh(script: "set +x && kubectl create secret generic ${secret_name} --from-literal='${key_name}'='${json_value}' --namespace=${namespace}")
+    } catch (Exception e) {
+        currentBuild.result = 'UNSTABLE'
+        println(e.getMessage())
+    }
```

Comment:
For these operations we don't need to try catch, all exceptions should be handled at higher stages.